crates/ide/src/rename.rs

```diff
@@ -1433,7 +1422,7 @@ pub fn baz() {}
                         ),
                         [
                             Indel {
-                                insert: "r#fn",
+                                insert: "fn",
```

Comment:
This is wrong no?

Reply:
Oops, forgot to handle the renaming of `mod` in the definition (it is in its own function).